framework/include/base/MooseApp.h

```diff
@@ -144,12 +139,6 @@ class MooseApp :
   std::list<std::string> getCheckpointFiles();
 
   /**
-   * Whether or not an output position has been set.
-   * @return True if it has
-   */
-  bool hasOutputPosition() { return _output_position_set; }
```

Comment:
But we don't care whether or not anybody has explicitly called the setOutputPosition function. - we just output at the offset no matter what.  If the offset is zeros, then fine.